documentation/src/main/asciidoc/topics/con_jvm_heap_memory_allocation.adoc

```diff
@@ -1,26 +1,100 @@
 [id='jvm-heap-memory-allocation_{context}']
 = JVM heap space allocation
 
-When allocating JVM heap space for {brandname} deployments, you can use the following guidelines:
+Determine the amount of JVM memory that is required for your {brandname} deployment, so that you have sufficient data storage capacity to meet your needs.
+
+[IMPORTANT]
+====
+Allocating too much of JVM heap space for data storage can lead to more frequent garbage collection (GC) and GC pauses.
+These operations can lead to {brandname} cluster instability and network partitions.
+====
+
+The following table outlines two examples of JVM heap space allocation scenarios:
```

Comment:
I would mention that the % is a rule of thumb which should be save to run a cluster, but depending on the data and usage pattern it might be different. Also it can change over the time if the data/pattern will change - i.e. increase throughput

Reply:
Thanks, @wfink . Updated the lead-in sentence and added a note under the table.